### ISSUE TITLE
SeedS3Data CFN resource takes 40+ mins (unresolved)

### ISSUE THREAD

**Issue by troydieter** [aws-samples/public-file-browser-for-amazon-s3#5]
Hi-

I'm seeing the `SeedS3Data` resource fail to deploy, as it's 40+ minutes:

<img width="1006" alt="image" src="https://github.com/aws-samples/public-file-browser-for-amazon-s3/assets/3689896/d5eea7a8-1729-4cb4-a559-3b3376092383">

**Comment by ckabalan** [comment]
Take a look at the recent CloudWatch Logs for the CustomSeedS3Data Lambda function (in that screenshot). If you can share those here I can take a deeper look. In short this function should complete within ~10sec and just copies the website code into the deployed bucket. I suspect there was a race condition or some other error, maybe an edge case with your input parameters. Either way I'd like to take a look and improve the code.

**Comment by troydieter** [comment]
My apologies @ckabalan - I totally overlooked the `./sam/DEPLOYMENT.md` file and didn't build the SAM project first. As you can see where this is going.. the function was faulting on the ability to import the `crhelper`. 

TL;DR: Read the docs, and it worked!
